cms/articles/models.py

```diff
@@ -166,6 +176,25 @@ class StatisticalArticlePage(BundledPageMixin, RoutablePageMixin, BasePage):  #
     dataset_sorting = models.CharField(choices=SortingChoices.choices, default=SortingChoices.AS_SHOWN, max_length=32)
     datasets = StreamField(DatasetStoryBlock(), blank=True, default=list)
 
+    # Featured chart for Topic page display
+    featured_chart = StreamField(
```

Comment:
suggestion: move these to a blocks module

Reply:
I have done this in 60d1b260f06bce4c2a5d9269595b999cd2937a7c.